packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/PipelineRequestHeadersExtensionsDefinition.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ClientModel.Primitives;
```

Comment:
I would expect to see this in the internal folder for Unbranded-TypeSpec local library?  Why isn't it there?

Reply:
hello @m-nash  I regened the Unbranded-TypeSpec, the file is generated there.